### PR TITLE
Enhance Maven Build and Prepare For Maven Central Deployment

### PR DESCRIPTION
**Pull request by ferstl**
This PR makes some cleanups and simplifications in the Maven build and enhances it to meet the [requirements for Maven Central deployment](http://central.sonatype.org/pages/requirements.html).

The main simplifications are:
- The JDK profiles do only define the paths to tools.jar (if available) and jfxrt.jar
- One global and simplified configuration of the exec-maven-plugin
- The `MANIFEST.MF` will now be generated during the build. The file in `src/main/resources/META-INF` is not necessary anymore.

Enhancements for Maven Central deployment:
- Add a `<distributionManagement>` section pointing to the Sonatype snapshot and staging repositories
- Complete the missing parts in the POM
- Create source and Javadoc JARs (Javadoc only during release build)
- Create PGP signatures during release build (PGP needs to be configured for that)

What still needs to be done for Maven Central deployment:
Basically, you need to get access to the Sonatype repositories and setup GPG. There is plenty of documentation for the Maven Central deployment. The main entry point is the [OSSRH Guide](http://central.sonatype.org/pages/ossrh-guide.html). The instructions for Maven can be found [here](http://central.sonatype.org/pages/apache-maven.html).

### CODE REVIEW

**Comment by chriswhocodes** [comment]
Thanks for the pom cleanup Stefan :)
